.release/docker/dockerfile

```diff
@@ -0,0 +1,84 @@
+#This is the docker official image
```

Comment:
Nit: We might want to link to https://hub.docker.com/_/vault and https://docs.docker.com/docker-hub/official_images/  :-)